doc/layer1_refutation_game.md

```diff
@@ -0,0 +1,216 @@
+[Original hackmd](https://hackmd.io/_Gl1CuggQ1GiITy-giLuCg)
+
+# Chusai Refutation game
```

Comment:
block-proposer / challenger might not be clear enough, the refutation dispute will (or might ?) probably involve two block proposer. I used game metaphor as we were talking about "refutation game" since early on, but I agree that naming is important. 
@xvw @rueshyna @Hakimba what do you think ?

Reply:
- `game` is ok for me. It's common used term in optimistic rollup.
- `split`: I'd prefer `dissection` but `split` is ok for me. It should not be `disagree`.  It's a series of consecutive segments, not about `disagree`.
- `attack`: I'd like to call `challenge`. 
- `player A` / `player B`: I agree with @pecornilleau

Reply:
> * `split`: I'd prefer `dissection` but `split` is ok for me. It should not be `disagree`.  It's a series of consecutive segments, not about `disagree`.

 I used `split` to avoid comitting to having two parts (could be any number in general, I started with 2, but I think I heard GA talk about 3 being the sweet spot at some point)

Reply:
Since `game` is pretty common, I suggest to keep it. 
I like `dissection` instead of `bissection` because we can imagine a world where `chusai` will be improved to deal with n-step dissection.
Instead of `attacker`, what do you think about `refuter`?

Reply:
The problem with `refuter` (and `attacker` even maybe) is that the roles alternate between the players. The take turn being on the offence so to speak. 
`refute` also sounds to me like a complete and final operation, not a turn in a game.

Reply:
I changed {